sdk/eventhub/Azure.Messaging.EventHubs.CheckpointStore.Blobs/tests/Azure.Messaging.EventHubs.CheckpointStore.Blobs.Tests.csproj

```diff
@@ -32,4 +32,7 @@
   <ItemGroup>
     <Compile Include="..\..\Azure.Messaging.EventHubs\tests\Shared\*.cs" Link="SharedSource\%(Filename)%(Extension)" />
   </ItemGroup>
+  
+  <!-- Import the references to the Azure client libraries -->
+  <Import Project="$(MSBuildThisFileDirectory)..\Azure.Client.Library.References.props" />
```

Comment:
Unfortunately, we can't use a direct project reference that crosses into a package that we reference.  Otherwise, as we found out painfully a short while ago, we get a mix of package and project references to the same library which really upsets the build.  

We'll have to come up with something to work around this.  The best thought I have at the moment is to link the source in, but I'm not a fan of that ,so... I shall take some time to ponder.